alacritty/src/renderer/mod.rs

```diff
@@ -1034,6 +1034,7 @@ impl<'a> RenderApi<'a> {
 
     pub fn render_cell(&mut self, cell: RenderableCell, glyph_cache: &mut GlyphCache) {
         let chars = match cell.inner {
+            RenderableCellContent::Cursor(cursor_key) if cursor_key.blinking_out => return,
```

Comment:
Isn't this a bit late? We could save ourselves a lot of work if we just filtered it out much earlier.